plugins/modules/ipagroup.py

```diff
@@ -393,13 +393,13 @@ def main():
                         # If yes: modify
                         # Also if it is a modification from nonposix to posix
                         # or nonposix to external.
-                        if not compare_args_ipa(ansible_module, args,
-                                                res_find) or \
-                           (
-                               not is_posix_group(res_find) and
-                               not is_external_group(res_find) and
-                               (posix or external)
-                           ):
+                        if not compare_args_ipa(
+                                ansible_module, args, res_find
```

Comment:
I think it would be better to use 4 spaces here, so it aligns with the second expression.